### ISSUE TITLE
Functional test failures in the sys.config tests

### ISSUE THREAD

**Issue by pjbreaux** [F5Networks/f5-common-python#437]
The functional test to save the config for the sys organizing collection is failing as shown below. The module with the issue is likely: f5/bigip/tm/sys/config

Here's the error:

=========================================================== test session starts ============================================================
platform linux2 -- Python 2.7.6, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
rootdir: /root/common, inifile:
plugins: symbols-0.1.0-a3, cov-2.2.1
collected 1 items

test_config.py F

================================================================= FAILURES =================================================================
___________________________________________________________ TestConfig.test_save ___________________________________________________________

self = <test.functional.tm.sys.test_config.TestConfig object at 0x7fe5f1f23050>, bigip = <f5.bigip.BigIP object at 0x7fe5f1f230d0>

```
def test_save(self, bigip):
    c = bigip.sys.config
```

> ```
>   c.save()
> ```

test_config.py:20:

---

container = <f5.bigip.tm.sys.config.Config object at 0x7fe5f1f23650>, name = 'save'

```
def __getattr__(container, name):
    # ensure this object supports lazy attrs.
    cls_name = container.__class__.__name__
    if 'allowed_lazy_attributes' not in container._meta_data:
        error_message = ('"allowed_lazy_attributes" not in',
                         'container._meta_data for class %s' % cls_name)
        raise LazyAttributesRequired(error_message)

    # ensure the requested attr is present
    lower_attr_names =\
        [la.__name__.lower() for la in
            container._meta_data['allowed_lazy_attributes']]
    if name not in lower_attr_names:
        error_message = "'%s' object has no attribute '%s'"\
            % (container.__class__, name)
```

> ```
>       raise AttributeError(error_message)
> ```
> 
> E           AttributeError: '<class 'f5.bigip.tm.sys.config.Config'>' object has no attribute 'save'

../../../../f5/bigip/mixins.py:95: AttributeError
========================================================= 1 failed in 0.04 seconds =========================================================
root@836b3b0120ae:~/common/test/functional/tm/sys#

**Comment by zancas** [comment]
Hmmm...

**Comment by caphrim007** [comment]
@pjbreaux i noticed the same

**Comment by pjbreaux** [comment]
Sorry that output sucks. Does anyone know how to paste it with no formatting?

**Comment by zancas** [comment]
@pjbreaux here's the potato

**Comment by wojtek0806** [comment]
@zancas 

Since we changed command mixin now the method of calling config save/load will look like this:

bigip.tm.sys.config.exec_command('save')
bigip.tm.sys.config.exec_command('load')

**Comment by wojtek0806** [comment]
@pjbreaux I had to remove 'load' options due some unexpected problems, was getting 503 error responses from bigip about MCP:

```
{"code":503,"message":"Fatal MCP error -- terminated","errorStack":[]}
```

**Comment by wojtek0806** [comment]
The config element needs some rework later, in conjunction with other improvements of doing BIGIP backups via REST. Another thing on to do list i guess

**Comment by pjbreaux** [comment]
Oh yeah, we should also think having unit tests to sniff out these simple oversight bugs. Then they would be caught by a build failure when you create the PR. Although they are not substitutes for a good functional test. It's up to you how to balance that. There are some good examples of both.

**Comment by wojtek0806** [comment]
fixed in 0.1.7
